http/notification_rule.go

```diff
@@ -163,21 +163,41 @@ type notificationRuleResponse struct {
 	LastRunError    string                `json:"LastRunError,omitempty"`
 }
 
+type ruleResponseMeta struct {
+	Labels          []influxdb.Label      `json:"labels"`
+	Links           notificationRuleLinks `json:"links"`
+	Status          string                `json:"status"`
+	LatestCompleted time.Time             `json:"latestCompleted,omitempty"`
+	LatestScheduled time.Time             `json:"latestScheduled,omitempty"`
+	LastRunStatus   string                `json:"lastRunStatus,omitempty"`
+	LastRunError    string                `json:"lastRunError,omitempty"`
+}
+
+func (resp *notificationRuleResponse) UnmarshalJSON(v []byte) (err error) {
```

Comment:
Note for reader:
The unmarshalling here never worked. Leading this client to result in a `nil` NotificationRule and a `nil` error when positively finding a rule.

Reply:
This was causing a panic when this client was used for pkger tests.